source/educators/how-tos/advanced_features/configure_content_experiments.rst

```diff
@@ -1,29 +1,16 @@
 .. _Configure Your Course for Content Experiments:
 
-#####################################################
 Configure Your Course for Content Experiments
-#####################################################
+#############################################
 
 .. tags:: educator, how-to
 
-This section provides instructions for configuring your course to use
-:ref:`content experiments<Overview of Content Experiments>`.
+This how-to provides instructions for configuring your course to enable
+:ref:`content experiments<Overview of Content Experiments>`. Let's start.
 
-.. contents::
-  :local:
-  :depth: 1
 
-.. _Enable Content Experiments:
-
-****************************************
-Enable Content Experiments
-****************************************
 
-To enable content experiments in your course, you add ``split_test`` to the
-**Advanced Module List** in Advanced Settings.
-
-.. note::
```

Comment:
Why delete this note, and the L22? It might not be obvious how the phrase split_test relates to experimnets, and L22 tells in advance what the steps will walk you through

Reply:
This information is passed to the specific How-to create_content_experiment.rst (source/educators/how-tos/advanced_features/create_content_experiment.rst). I had to break this how-to into several documents, but I can add that information here again if necessary.

Reply:
I would add these back here in this place. I don't think it's good to expect that someone reads through every related how-to article in order to fully understand a component.

Reply:
@Apgomeznext - once this comment is addressed we can merge this PR.

Reply:
Comment addressed.